src/customJs/components/EmptyViewer.test.js

```diff
@@ -0,0 +1,45 @@
+import { React } from '../unlayer-react';
+import { render, screen } from '@testing-library/react';
+import { EmptyViewer } from './EmptyViewer';
+
+describe(EmptyViewer.name, () => {
+  const rest = {
+    isViewer: true,
+    toolInfo: {
+      icon: 'toolIconPath',
+      label: 'toolLabel',
+      name: 'toolName',
+    },
+  };
+
+  it('must be render the icon', async () => {
```

Comment:
We use the word _should_ instead of _must_ in these projects (very minor).

Also, if I am not wrong, the test name should be readable replacing `it` with the _"describe's subject"_, in this case, `EmptyViewer`. So, in place of _"EmptyViewer must be render the icon"_ should be something like _"EmptyViewer must render the icon"_.

```suggestion
  it('should render the icon', async () => {
```